### DISCUSSION TITLE
i wan't use go to create container for host network

### DISCUSSION THREAD

**Discussion by wike2019** [moby/moby#44543] [Q&A]
if use shell i can create container with host network use this command .
docker run --restart=always -d --network host nginx
but i can't use go to create container like shell, who can help me .

**Comment by wike2019** [comment]
var c = &container.Config{
		Image: "nginx",
	}
	h := &container.HostConfig{NetworkMode: "host"}
	ctx := context.Background()
	cli.ContainerCreate(ctx, c,  h, &network.NetworkingConfig{} ,&v1.Platform{},"test")


this is my code

**Comment by helmutkemper** [comment]
https://github.com/helmutkemper/chaos/blob/main/internal/builder/networkCreate_test.go
